Makefile

```diff
@@ -2,8 +2,10 @@ include .bingo/Variables.mk
 
 .DEFAULT_GOAL=textile
 
-textile:
-	go install ./...
+GOVVV_FLAGS=$(shell $(GOVVV) -version git -flags -pkg $(shell go list ./buildinfo))
```

Comment:
okay, 

default version is `development` for `go install ./...`. running `make textile` changes it to `git` so that i can fully display the git info in the version command results easily. screenshots coming